src/common/dcgraph.cpp

```diff
@@ -752,9 +752,16 @@ void wxGCDCImpl::DoDrawPoint( wxCoord x, wxCoord y )
     if (!m_logicalFunctionSupported)
         return;
 
-#if defined(__WXMSW__) && wxUSE_GRAPHICS_GDIPLUS
-    // single point path does not work with GDI+
-    if (m_graphicContext->GetRenderer() == wxGraphicsRenderer::GetGDIPlusRenderer())
+#if defined(__WXMSW__)
+    bool useStrokeLine = false;
+#if wxUSE_GRAPHICS_GDIPLUS
```

Comment:
At this point (pun not intended), I'd rather add a `virtual CanStrokeSinglePointLine()` or something like this to `wxGraphicsRenderer`.

Reply:
This still wouldn't cover the difference between macOS and Cairo.
Maybe I should add `wxGraphicsContext::DrawPoint()`?

Reply:
Yes, this is probably an even better idea.

Reply:
I found an implementation (using `AddCircle`) that works for all renderers.